scss/_progress.scss

```diff
@@ -28,52 +28,52 @@
   // Set overall border radius
   @include border-radius($border-radius);
 }
+
+// Filled-in portion of the bar
 .progress[value]::-ms-fill {
   background-color: $progress-bar-color;
   // Remove right-hand border of value bar from IE10+/Edge
   border: 0;
 }
-.progress[value]::-webkit-progress-bar {
-  background-color: $progress-bg;
-  @include border-radius($border-radius);
-  @include box-shadow(inset 0 .1rem .1rem rgba(0,0,0,.1));
+.progress[value]::-moz-progress-bar {
+  background-color: $progress-bar-color;
+  border-top-left-radius: $border-radius;
+  border-bottom-left-radius: $border-radius;
 }
 .progress[value]::-webkit-progress-value {
   background-color: $progress-bar-color;
   border-top-left-radius: $border-radius;
   border-bottom-left-radius: $border-radius;
 }
+// Tweaks for empty progress bar
+.progress[value="0"]::-moz-progress-bar {
+  color: $gray-light;
```

Comment:
Properties should be ordered min-width, color, background-color, background-image